lib/parser/parser-expr.h

```diff
@@ -52,6 +52,8 @@ log_parser_deinit_method(LogPipe *s)
   return TRUE;
 }
 
+EVTTAG *parser_result_tag(gboolean res);
```

Comment:
Does this need to be part of the public interface? I think this function can be static. Similar comment to the other two variants.

Reply:
Agreed, I will change this.

Reply:
Fixed.